src/doc/man/options-token.adoc

```diff
@@ -1,2 +1,8 @@
 *--token* _TOKEN_::
-    API token to use when authenticating.
+    API token to use when authenticating. This overrides the token stored in
+    the credentials file (which is created by man:cargo-login[1]).
++
+The token for crates.io may be specified with the `CARGO_REGISTRY_TOKEN`
+environment variable. Tokens for other registries may be specified with
+environment variables of the form `CARGO_REGISTRIES_NAME_TOKEN` where `NAME`
+is the name of the registry in all capital letters.
```

Comment:
I wonder if this could perhaps mention that the env vars are derived from the `.cargo/config` configuration to ensure both mechanisms are mentioned here?